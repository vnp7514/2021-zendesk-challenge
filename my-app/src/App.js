import logo from './logo.svg';
import './App.css';
import { QueryClient, QueryClientProvider } from 'react-query';
import TicketViewer from './TicketViewer';

// Create a client
const queryClient = new QueryClient();
function App() {
  return (
    <div className="App">
      <QueryClientProvider client={queryClient}>
        <TicketViewer />
      </QueryClientProvider>
    </div>
  );
}

export default App;
