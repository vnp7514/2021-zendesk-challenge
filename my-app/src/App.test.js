import { render, screen } from '@testing-library/react';
import App from './App';

test('Application starts with 1 button', () => {
  const {container} = render(<App />);
  expect(container).toMatchSnapshot();
});
