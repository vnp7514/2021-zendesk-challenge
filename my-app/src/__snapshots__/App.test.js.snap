// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application starts with 1 button 1`] = `
<div>
  <div
    class="sc-jRQBWg fVRA-DT"
  >
    <p />
    <p />
    <p />
    <button
      data-testid="gettickets"
    >
      Fetch Data
    </button>
  </div>
</div>
`;
